hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -6151,4 +6151,14 @@ public final boolean isAllNull(Object[] array, int tableNumber) {
 		}
 		return true;
 	}
+
+	@Override
+	public Loadable asLoadable() {
```

Comment:
this might work, but I'd have preferred to add such overrides as default methods in the interface? Was there a reason to not do so?

That would also have avoided the need to implement them in the various test implementations.

Reply:
Yes the reason is the Loadable instance type is not available at the EntityPersister interface implementation. At extended interfaces it is. Attempting this causes the compiler to report this as an error.

> error: incompatible types: EntityPersister cannot be converted to Loadable